scripts/src/language.ts

```diff
@@ -0,0 +1,44 @@
+import acceptLanguage from "accept-language";
+
+acceptLanguage.languages(["en", "pt"]);
+const language: string = acceptLanguage.get(window.navigator.language) || "en";
+
+export type Language = {
```

Comment:
Can you satisfy my OCD and change this to `LocalizedStrings` or `LocalizationDictionary` or something

Reply:
done